app/components/m-header.hbs

```diff
@@ -1,87 +1,160 @@
-              @current-when="search"
+                    @route="search.index"
+                    @current-when="search"
             >
-              {{!-- TODO: magnifying glass icon --}}
-              <span class="auk-tabs__tab-inner">{{capitalize (t "search")}}</span>
+              <span class="auk-tabs__tab-inner">
+                <div class="auk-icon auk-icon--default auk-icon--search">
+                  <i class="ki ki-search"></i>
+                </div>
+                <span class="auk-tabs__label">{{capitalize (t "search")}}</span>
+              </span>
             </LinkTo>
           </li>
-          {{#if currentSession.isEditor}}
+          {{#if this.currentSession.isEditor}}
             <li class="auk-tabs__tab" data-test-m-header-settings>
-              {{#link-to "settings.overview"}}
-                <span class="auk-tabs__tab-inner">{{t "settings"}}</span>
-              {{/link-to}}
+              <LinkTo @route="settings.overview">
+                <span class="auk-tabs__tab-inner">
+                  <span class="auk-tabs__label">
+                    {{t "settings"}}
+                  </span>
+                </span>
+              </LinkTo>
             </li>
           {{/if}}
         </ul>
       </div>
     </div>
     <div class="auk-toolbar-complex__right">
-      <div class="auk-toolbar-complex__right">
-        {{#if showEnvironmentName}}
-          <div class="vlc-environment-pill {{environmentClass}}">
-            {{environmentName}}
-            {{capitalize (t "environment")}}
+      {{#if this.showEnvironmentName}}
+        <div class="auk-toolbar-complex__item auk-u-mr-3">
+          <div class="vlc-environment-pill {{this.environmentClass}}">
+            {{this.environmentName}}
           </div>
-        {{/if}}
-      </div>
-      <div class="auk-toolbar-complex__item">
-        <div class="auk-u-flex auk-u-flex--vertical-center auk-u-text-small auk-u-mx-2">
-          {{#if currentSession.userContent}}
-            <div class="auk-u-mr">
-              {{currentSession.userContent.firstName}}
-              {{currentSession.userContent.lastName}}
-              ({{currentSession.userRole}})
-            </div>
-            <a href="" {{action "logout"}}>
-              {{t "sign-out"}}
-            </a>
-          {{else}}
-            {{#link-to "login"}}
-              {{t "login"}}
-            {{/link-to}}
-          {{/if}}
         </div>
+      {{/if}}
+      <div class="auk-toolbar-complex__item">
+        <WebComponents::AuButtonLink
+                @skin="muted"
+                @icon="ki ki-circle-question">
+          {{t "help"}}
+        </WebComponents::AuButtonLink>
+        <AttachPopover
+                @renderInPlace={{true}}
+                @class="ember-attacher-popper"
+                @hideOn="clickout click"
+                @showOn="click"
+                @animation="shift"
+                @placement="bottom" {{!-- as |attacher| --}}
```

Comment:
@placement="bottom-end" is wat je hier wilt, dan lijkt het meer het design (whitespace tussen rechterkant en je dropdown menu)